### PR TITLE
Add anolis 8.4 docker image support

### PR DESCRIPTION
**Pull request by ABILLEST**
Add a dockerfile named 'Dockerfile.anolis8.4' for building occlum image for Anolis 8.4.
1. The SGX SDK version in the docker file has been updated to 2.17.
2. Notice is commented at the top line of the docker file that anolis is not an official supported distribution now.

### CODE REVIEW

File: tools/docker/Dockerfile.anolis8.4 [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+RUN cd glibc && ./build.sh && rm -rf /tmp/glibc
+
+# Install Occlum Golang toolchain
+WORKDIR /tmp
+RUN cd golang && ./build.sh && rm -rf /tmp/golang
+ENV PATH="/opt/occlum/toolchains/golang/bin:$PATH"
+
+# Install Rust
+ENV PATH="/root/.cargo/bin:$PATH"
+ENV OCCLUM_RUST_VERSION=nightly-2021-11-01
```
Comment (qzheng527) [inline tools/docker/Dockerfile.anolis8.4:152]: Please use "nightly-2022-02-23".

File: tools/docker/Dockerfile.anolis8.4 [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+RUN cd java && ./install_dragonwell.sh && rm -rf /tmp/java
+ENV PATH="/opt/occlum/toolchains/jvm/bin:$PATH"
+
+# Install OpenJDK 8
+ARG JDK8_PATH=/opt/occlum/toolchains/jvm/java-1.8-openjdk
+COPY --from=alpine /usr/lib/jvm/java-1.8-openjdk $JDK8_PATH
+RUN rm $JDK8_PATH/jre/lib/security/cacerts
+COPY --from=alpine /etc/ssl/certs/java/cacerts $JDK8_PATH/jre/lib/security/cacerts
+
+ENV LD_LIBRARY_PATH="/opt/intel/sgxpsw/aesm:/usr/lib64/:/usr/local/lib:/opt/intel/sgxsdk/sdk_libs:/usr/lib:$LD_LIBRARY_PATH"
```
Comment (qzheng527) [inline tools/docker/Dockerfile.anolis8.4:186]: Is it necessary to set LD_LIBRARY_PATH here? Part of them will be done in the following "source /opt/intel/sgxsdk/environment".

File: tools/docker/build_image.sh [hunk 3]
```diff
@@ -23,6 +23,7 @@ USAGE:
         ubuntu20.04         Use Ubuntu 20.04 as the base image
         centos8.2           Use CentOS 8.2 as the base image
         aliyunlinux3        Use AliyunLinux 3 as the base image
+	    anolis8.4	        Use Anolis 8.4 as the base image
```
Comment (qzheng527) [inline tools/docker/build_image.sh:26]: Please make it space aligned.

**Comment by qzheng527** [comment]
LGTM

**Review by volcano0dr — APPROVED** [review]
LGTM.
@ABILLEST Thanks for your contribution!

**Pull request merged**